### PR TITLE
add style and content

### PR DESCRIPTION
**Pull request by Empino27**
## These are the modified and added contents
- Added Banner
- List of games

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,19 +2,26 @@
...
-	<ul>
-		<li>Jayrald</li>
-<li>Bulleser</li>
-<li>Empino</li>
-	</ul>
+	<main class="experiences">
+		<h2>Experiences:</h2>
+		<ul>
+			<li>Solve mystery puzzles through API</li>
+			<li></li>
```
Comment (Jayrald07) [inline index.html:21]: Please complete this

Comment (Empino27) [reply index.html:21]: Okay. I'll get back

Comment (Jayrald07) [reply index.html]: **Done.**

**Review by Jayrald07 — APPROVED** [review]
It's all good.

**Pull request merged**